heudiconv/bids.py

```diff
@@ -487,12 +486,34 @@ def convert_sid_bids(subject_id):
     subject_id : string
         Original subject ID
     """
+    sid, subject_id = sanitize_label(subject_id)
+    lgr.warning('Deprecation warning: convert_sid_bids() is deprecated, '
+                'please use sanitize_label() instead.')
```

Comment:
wow -- apparently we never issued any deprecation warning!  for those we should not use logger but rather warnings system.  

```shell
import warnings
...
warnings.warn("Message", DeprecationWarning)
```